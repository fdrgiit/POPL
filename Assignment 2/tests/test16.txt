a b c d e f #
L R I S N A G D C V U W #
a -> L b R #
a -> d e #
b -> #
b -> b I c S #
b -> b N c S #
b -> b A c S #
b -> b G c S #
c -> D #
c -> c C D #
d -> #
d -> V V U #
d -> W d #
e -> f e #
e -> f #
f -> #

z -> hello B L I #
z -> b G e #
j -> #
N -> #
I -> I R A N #
U -> U S A #
u -> a b c w y x #
##

