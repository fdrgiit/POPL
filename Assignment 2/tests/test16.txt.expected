ERROR CODE 2
ERROR CODE 3
